.github/workflows/test-docs-build.yml

```diff
@@ -0,0 +1,21 @@
+name: Test Docs Build
```

Comment:
Should it still be `Test`?

Reply:
Why not? It's still just a test. Mainly to detect broken links